json-exporter/config/config.yml

```diff
@@ -0,0 +1,224 @@
+# Reusable extensions (used for pushing data from different parts of the JSON to the same metric)
```

Comment:
Looks like we're missing the `seven_days` set of metrics under `messaging.outgoing`:
* `messaging.outgoing.seven_days.due`
* `messaging.outgoing.seven_days.scheduled`
* `messaging.outgoing.seven_days.muted`
* `messaging.outgoing.seven_days.delivered`
* `messaging.outgoing.seven_days.failed`

Reply:
Thanks for double-checking these metrics!  I intentionally left these out, but am open to discussing!  

Basically, my thinking was that we did not really need the `seven_days` metrics because, thanks to the power of Prometheus, we should be able to calculate all this data from `cht_messaging_outgoing_total` by focusing on just the last 7 days (or any other time range we wanted).

Reply:
@m5r currently we don't have any great way to auto-check that we are consuming all the data from `/monitoring`.    Ultimately, I think we want to move in the direction of https://github.com/medic/cht-core/issues/8124 which will let us drop the json-exporter layer altogether and then automatically be able to ingest all available metrics.